### PR TITLE
Bl or 604 mk

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient604.xml [hunk 1]
```diff
@@ -33,43 +34,234 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+    <msItem xml:id="ms_i1">
+    <locus from="5r"/>
+    <title type="complete" xml:lang="en" ref="LIT1293Dersan">Discourse of <persName ref="PRS9486Theophil">Theophilus of Alexandria</persName></title>
+    <incipit xml:lang="gez">ይቤ፡ በእንተ፡ እግዝእትነ፡ ማርያም፡ ድንግል፡ ወበእንተ፡ ቤት፡ ዘኀደረት፡ ውስቴቱ፡ ምስለ፡ ፍቁር፡ ወልዳ፡ ኢየሱስ፡ ክርስቶስ፡ በደብረ፡ ቍስቋም፡
+    ዘመልዕልተ፡ ደብረ፡ አመ፡ መጽኡ፡ ብሔረ፡ ግብጽ፡ እምፍርሃተ፡ ሄሮድስ፡ ንጉሥ። ወበእንተዝ፡ ወፅአ፡ ሊቀ፡ ጳጳሳት፡ ወበጽሐ፡ ወርእየ፡ ዘንተ፡ ኵሎ፡ ተአምረ። እስመ፡ ንጉሥ፡ በስጣዮስ፡ ክርስቲያናዊ፡
+    ወሀቦ፡ መራኁተ፡ ኵሉ፡ ቤተ፡ አማልክት፡ ዘብሔረ፡ ግብጽ፡ ከመ፡ ይንስቶሙ፤ ወይንሣእ፡ ኵሎ፡ ንዋዮሙ። ወይሕንፅ፡ አብያተ፡ ክርስቲያናት፡ ውስተ፡ ኵሉ፡ አድያሚሃ፡ ለግብፅ። ወሀለዉ፡ ኤጲስ፡ ቆጶሳት፡
+  ምስሌሁ። ወዜነውዎ፡ ክብረ፡ ለውእቱ፡ ደብር፡ ወጸጋ፡ ዘቤተ፡ ክርስቲያን፡ ዘላዕሌሁ። ወፈቀደ፡ ይሑር፡ ኀቤሁ፡ ከመ፡ ይንሣእ፡ በረከቶ፡ ወትንብልናሁ፡ ወበጽሐ፡ እንዘ፡ ዓዲ፡ ሠሉስ፡ መዋዕል፡ ለተዝካረ፡ በዓሉ፡
+  ለውእቱ፡ ደብረ። ወአስተብቍዕዎ፡ ኤጲስ፡ ቆጶሳት፡ ወመነኮሳት፡ ይንበር፡ ምስሌሆሙ፡ እስከ፡ የኃልፍ፡ በዓለ፡ ለቅድስት፡ ማርያም፡ ወኍልቆሙ፡ ፫፻ወነበረ፡ ምስሌሆሙ፡ በሰላመ፡ እግዚእነ፡ አሜን።</incipit>
+  <note>To be read on the 6th of Ḫǝdār</note>
+  <note>ዘይትነበብ፡ አመ፡ ፯ለወርኃ፡ ኅዳር፡ ዘከመ፡ ኀደረት፡ እግዝእትነ፡ ውስተ፡ ደብረ፡ ቍስቋም።</note>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient604.xml:51]: the element note is not appropriate here

File: LondonBritishLibrary/orient/BLorient604.xml [hunk 2]
```diff
@@ -33,43 +34,234 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+
+<item xml:id="a1">
+<desc type="MixedNote">The volume is full of lists of furniture, dress and books, deeds of gift, and similar documents, in Amharic, written
+on the blank pages and on the margins. They seem to belong chiefly to the reigns of <persName ref="PRS10473Yostos">Yosṭos</persName>,
+<persName ref="PRS5617IyasuII">ʾIyāsu II</persName>, <persName ref="PRS5636IyoasI">ʾIyoʾās I</persName>,
+<persName ref="PRS10376Yohanne">John II</persName>, <persName ref="PRS9152TaklaHa">Takla Hāymānot</persName>, <persName ref="PRS8421Salomon">Solomon</persName>, <persName ref="PRS9142TaklaGi">Takla Giyorgis</persName>, <persName ref="PRS5618IyasuII">Iyāsu III</persName>,
+<persName ref="PRS5332hezqeyas">Ḥǝzqǝyās</persName>, <persName ref="PRS3501Demetros">Dǝmeṭros</persName>, <persName ref="PRS3692egwalas">ʾƎgʷāla Ṣǝyon</persName>,
+ʾIyoʾās II, <persName ref="PRS4661Gigar">Gigār</persName>, <persName ref="PRS5619IyasuIV">Iyāsu IV</persName>, <persName ref="PRS4264GabraKr">Gabra Krǝstos</persName>, <persName ref="PRS8344sahlaDe">Śāhla Dǝngǝl</persName> and
+<persName ref="PRS9430Tewodros">Theodore</persName>.</desc>
+</item>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient604.xml:232]: Please, add an empty q with the language

File: LondonBritishLibrary/orient/BLorient604.xml [hunk 3]
```diff
@@ -33,43 +34,234 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<item xml:id="a2">
+<locus target="#5r"></locus>
+<desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+<placeName ref="INS0101MadhaneAlam"/>.</desc>
+<q xml:lang="gez">ነገረ፡ ማርያም፡ ዘ<placeName ref="INS0101MadhaneAlam"> ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName></q>
+</item>
+
+<item xml:id="e1">
+<desc>The name of the first owner has been erased, but the name of the king, in whose name the manuscript was written, is
+still legible, namely Dāwit e. g. on <locus target="#5r"> f. 5r</locus></desc>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient604.xml:243]: Please, add an id to the king

File: LondonBritishLibrary/orient/BLorient604.xml [hunk 4]
```diff
@@ -33,43 +34,234 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+<q xml:lang="gez">በረከቱ፡ ቅድስት፡ ተሃሉ፡ ምስለ፡ ንጉሥነ፡ <persName ref="PRS3430DawitII">ዳዊት፡</persName> ወምስለ፡ ገብሩ፡ <gap reason="omitted" resp="bm:Wright1877BM"/> ወምስለ፡
+ነፍሰ፡ ንጉሥነ፡ ኢያሱ፡ ለዓለመ፡ ዓለም፡ አሜን፡ ወአሜን።</q>
+</item>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient604.xml:246]: You cann add an id for Iyasu as well

File: LondonBritishLibrary/orient/BLorient604.xml (hunk 2 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient604.xml:232]: Is this what you meant?

File: LondonBritishLibrary/orient/BLorient604.xml (hunk 1 above)
Comment (MarcinKrawczuk) [reply LondonBritishLibrary/orient/BLorient604.xml:51]: What is your suggestion?

Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient604.xml:51]: Sth like: `<note>The instruction on the reading of the text on the 6th of Ḫǝdār and the title are written in the upper margin <foreign xml:lang='gez'><hi rend="rubric"><add place="top">ዘይትነበብ፡ አመ፡ ፯ለወርኃ፡ ኅዳር፡ ዘከመ፡ ኀደረት፡ እግዝእትነ፡ ውስተ፡ ደብረ፡ ቍስቋም።</add></hi><foreign/></note>`

File: LondonBritishLibrary/orient/BLorient604.xml (hunk 2 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient604.xml:233]: yes, but it should be am in accordance the description provided

**Review by DariaElagina — APPROVED** [review]

**Pull request merged**